Python/track_segmentation.py

```diff
@@ -48,9 +48,10 @@ def extract_features(audio_file, beats_file):
     """
 
     t = pd.read_table(beats_file, header=None)
-    beat_times = t.ix[:, 0].values
+    beat_times = t.iloc[:, 0].values
 
     beat_mls = compute_beat_mls(filename=audio_file, beat_times=beat_times)
+    beat_mls /= np.max(beat_mls)
```

Comment:
@mleimeister this is the only line I added that wasn't just part of "upgrade land"; brefore this track_segmentation.py was giving sort of nonsensical results.  I'm not exactly sure why...